### PR TITLE
Weakens the ridiculously armored windows

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
With the exception of shuttle windows, which got a small damage deflection making them immune to weak weapons.

Did you knew that plastitanium windows used to have 24,000 EHP against melee attacks?
Or that they could withstand 20 revolver shots.

It should now be more viable to break down reinforced windows with the use of toolboxes or spears, given enough time. (Compared to a ridiculous estimate of 500-1000 hits)
Reinforced plasma and plastitanium windows can now be broken with bullets if someone wants to dedicate around a magazine, compared to A COUPLE MAGS.

Besides, a person meta gaming trying to get from point A to B will hack a door or deconstruct a wall anyway. This gives people with weapons or guns more power.

This also makes getting unstuck from emergency situations not a total pain in the ass, if you have to break a reinforced window.

## Changelog
:cl:
balance: Weakens the ridiculously armored reinforced windows. Now melee attacks with strong weapons wont take ridiculously large time to break them open, and it's much more viable to break them open with gunshots.
/:cl:

### CODE REVIEW

**Review by atakiya — APPROVED** [review]
I'll TM this first so we can tweak it if something seems very off.
Shouldn't be in for more than one-two weeks however.

**Review by atakiya — APPROVED** [review]
TM pass.

I'm not going to keep this longer around to keep things fresh.
Just from testing one round this already makes interactions with windows much more viable.
There WILL be some more changes needed to balance around this for some things I would say, but that goes out of scope for this PR.

TL;DR - The armor values are good.
